### PR TITLE
docs: add of Björn Fahller's articles

### PR DESCRIPTION
**Pull request by crutchcorn**
> Preview Björn's profile: https://unicorn-utterances-git-add-bjorns-articles-oceanbit.vercel.app/unicorns/rollbear

This PR adds all of Björn Fahller's articles from his website "Playful Programming" to prep for our rebrand to "Playful Programming" itself (more soon).

Björn, a couple of open-ended questions for you:

1) Does everything here look okay to you? I gave the migration of your posts meticulous attention but wanted to make sure first
2) Can you modify your `index.md` and profile picture to-taste? (you can change it to anything, so long as it follows the format outlined here). You can do that by making a PR against this `add-bjorns-articles` branch. Feel free to rename your username as well (do this by moving the `rollbear` folder to a new folder)
3) Are you opposed to closing down your https://playfulprogramming.blogspot.com/ domain immediately prior to merging this PR?
    i) If you are, then we can remove the `originalLink` on each of the blog posts (I can do this for you)
    ii) If not, then we'll need to keep the `originalLink` on each article and you should probably not delete your blogspot down ever. However, with this it complicates our rename to "Playful Programming" as Googling would show up two search results.
    
I'd love to hear your thoughts; especially in regards to #3. I know we're asking you to make a leap of faith, but I think the end result would be a fruitful partnership going forward 💖

### CODE REVIEW

**Comment by crutchcorn** [comment]
CC @rollbear

File: content/rollbear/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	lastName: "Fahller",
+	description: "I write code, I do not drink coffee.\nAvatar by Franzisca Panter",
+	socials: {
+		twitter: "bjorn_fahller",
+		github: "rollbear",
+        mastodon: "https://mastodon.social/@bjorn_fahller"
+    },
+	pronouns: "",
+	profileImg: "./rollbear.jpg",
+	color: "#000000",
```
Comment (crutchcorn) [inline content/rollbear/index.md:14]: Change this to your favorite color to add a bit of customization in the future! 😊

File: content/rollbear/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	name: "Björn Fahller",
+	firstName: "Björn",
+	lastName: "Fahller",
+	description: "I write code, I do not drink coffee.\nAvatar by Franzisca Panter",
+	socials: {
+		twitter: "bjorn_fahller",
+		github: "rollbear",
+        mastodon: "https://mastodon.social/@bjorn_fahller"
+    },
+	pronouns: "",
```
Comment (crutchcorn) [inline content/rollbear/index.md:12]: I thought I'd seen your pronouns on one of your profiles, but wasn't sure after double-checking.

Feel free to remove this flag if you're uncomfortable disclosing

Comment (rollbear) [reply content/rollbear/index.md:12]: he/him is the right pronouns.

Drop twitter. As Kevlin Henney said, "twitter is now my ex". The last few posts I made there went into the void without any response at all, so it seems like the people who follow me there aren't logging in.

**Review by rollbear — CHANGES_REQUESTED** [review]
It's looking good. I only sampled a few of the articles, but they look good. You may merge after the suggested minor changes.

**Comment by rollbear** [comment]
>     3. Are you opposed to closing down your https://playfulprogramming.blogspot.com/ domain immediately prior to merging this PR?

I'm not sure what's the best way to do this. I want to offer some forwarding link to this site, so that people with book marks can find it.

Maybe make a last post saying "we've moved, go here for new contents"?

**Comment by crutchcorn** [comment]
> I'm not sure what's the best way to do this. I want to offer some forwarding link to this site, so that people with book marks can find it.

Actually, I created a Blogger/Blogspot account to see how we could smooth this migration and I think there's a good way to do this:

See the problem is that we can't have Google crawl _both_ your site and our site alike. Otherwise, it'll see us as the duplicate and deindex our site from the internet.

The way we can solve this is either:

1) Treat your `blogspot` links as the original and link back to your site programmatically (as we have here), but don't show your articles from Unicorn Utterances/the new Playful Programming domain

2) Treat the Unicorn Utterances/the new Playful Programming domain as the "original" by removing your site from being indexed by Google.

We can do that by going into your Blogger/Blogspot settings and changing this to disabled:

image

This to enabled:

image

With these options:

```
# *
User-agent: *
Disallow: /
```

Once this is done, we can leave your Blogspot as-is and turn on the Search Engine indexing for the new article links (our domain)

What do you think about plan #2?

**Comment by rollbear** [comment]
> What do you think about plan #2?

Done. Turned off visibility to search engines and enabled the disallowing robots.txt.

**Review by rollbear — APPROVED** [review]
Thanks

**Pull request merged**